introduction/why_c.md

```diff
@@ -1,14 +1,15 @@
-## なぜ C\# か？ {#why-c}
+# なぜ C\# か？ {#why-csharp}
```

Comment:
Will PR to original English version?

Reply:
@hypergori Understood. For now I changed them back and once I pr for English version I will revisit this again.